From http://xenbits.xen.org/hg/linux-2.6.18-xen.hg/rev/e8c8784ca274
From: xen-devel@lists.xenproject.org
Patch-mainline: n/a
Subject: x86-64/MCE: flip CPU and bank numbers in log message

Seeing "CPU8, BANK0" when "CPU0, BANK8" is meant can be rather
confusing when trying to decode the remaining printed information or
determine the offending physical CPU package. Reformat the message so
that one can recognize whether it's a properly or wrongly ordered one
without needing to know the precise kernel version.

Signed-off-by: Jan Beulich <jbeulich@suse.com>

--- sle11sp3.orig/arch/x86/kernel/cpu/mcheck/mce_dom0.c	2015-01-16 14:55:01.000000000 +0100
+++ sle11sp3/arch/x86/kernel/cpu/mcheck/mce_dom0.c	2014-12-08 13:19:31.000000000 +0100
@@ -38,8 +38,8 @@ static int convert_log(struct mc_info *m
 			m.tsc = mc_bank->mc_tsc;
 			m.res1 = mc_bank->mc_ctrl2;
 			m.bank = mc_bank->mc_bank;
-			printk(KERN_DEBUG "[CPU%d, BANK%d, addr %llx, state %llx]\n", 
-						m.bank, m.cpu, m.addr, m.status);
+			printk(KERN_DEBUG "[CPU %d bank %d addr %Lx state %Lx]\n",
+			       m.cpu, m.bank, m.addr, m.status);
 			/*log this record*/
 			mce_log(&m);
 		}
